Debian repositories #16335 keichwa (#1296)
uyuni-project/uyuni-docs: modules/administration/pages/custom-channels.adoc

. OPTIONAL: Complete the SSL fields if your repository requires client certificate authentication.
. Click btn:[Create Repository].

[NOTE]
====
Add Debian non-flat repositories using ``uyuni_suite``, ``uyuni_component``, and ``uyuni_arch`` query parameters.
====

``uyuni_suite``::
is mandatory.
With this parameter you specify the apt source.

> Without this parameter the original approach is used.

Author: ok, I'm now mention it in my local tree.

``uyuni_suite``::
is mandatory.
In Debian documentation, this is also known as [literal]``distribution``.
With this parameter you specify the apt source.

> Without this parameter or if it ends with a ``/``, the original approach is used (= flat repository).

Reviewer: No, it's not true. The removed line was correct. Additional note about `/` at the end of `uyuni_suite` is a separate case when the repo is defined as a flat. The previous approach is trying to detect flat and non-flat repos by presence of Release file in certain place of hierarchy. (but not sure if we need to explain it. So:
```suggestion
Without this parameter the original approach is used.
If the parameter ends with ``/``, the repository is identified as flat.
```

Author: yes, now I see.
